src/Microsoft.VisualStudio.Services.Agent/SecretMask.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.VisualStudio.Services.Agent
+{
+    public interface ISecretMask
```

Comment:
will rename the classes as suggested. I'm not sure what was the original idea to make these classes public instead of "AddRegEx", "AddValue". I think from OO viewpoint it is more extendable: you can have 100 mask classes in 100 separate files with separate unit tests, versus one big SecretMasker with 100 methods.